app/models/commitment.rb

```diff
@@ -10,7 +10,13 @@ class Commitment < ApplicationRecord
   import_by objectives: :name
   has_and_belongs_to_many :governance_types
   import_by governance_types: :name
+  
+  has_many :progress_documents
 
+  has_one_attached :spatial_data
+  validates :spatial_data, 
+    content_type: %w(application/vnd.google-earth.kml+xml application/vnd.google-earth.kmz application/zip), 
+    size: { less_than: 25.megabytes }
```

Comment:
No space between `has_one_attached` and the related `validates` here, in contrast with the spacing in `progress_document`